packages/katello/katello/katello-debug.sh

```diff
@@ -111,27 +108,24 @@ fi
 add_cmd "du -sh /var/lib/pgsql" "postgres_disk_space"
 add_cmd "du -sh /var/lib/mongodb" "mongodb_disk_space"
 add_cmd "df -h" "disk_space_output"
+add_cmd "du -sh /var/lib/qpid" "qpid_jrnl_disk_space"
+add_cmd "du -sh /var/lib/candlepin/hornetq" "hornetq_disk_space"
 
 # Proxy ENV Vars
 add_cmd "echo $http_proxy" "http_proxy_var"
 add_cmd "echo $https_proxy" "https_proxy_var"
 add_files /etc/profile.d/*
 
-# Qpid Debug
-add_cmd "qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-q"
-add_cmd "qpid-stat -u --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-u"
-add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-c"
-add_cmd "qpid-stat --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt  -b amqps://localhost:5671 -q resource_manager" "qpid-stat-resource_manager"
-add_cmd "ps -awfux" "ps-awfux"
-add_cmd "ps -efLm" "ps-elfm"
-add_cmd "rpm -qa | grep qpid" "qpid-rpm-qa"
 
 add_cmd "mongo pulp_database --eval \"db.reserved_resources.find().pretty().shellPrint()\"" "mongo-reserved_resources"
 add_cmd "mongo pulp_database --eval \"DBQuery.shellBatchSize = ${FOREMAN_DEBUG_MONGOTASKS:-200};; db.task_status.find().sort({finish_time: -1}).pretty().shellPrint()\"" "mongo-task_status"
 
 echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" > $TEMP_DIR/pulp_running_tasks.js
 add_cmd "mongo pulp_database $TEMP_DIR/pulp_running_tasks.js" "pulp-running_tasks"
 
+add_cmd "echo \"select id, name, checksum_type, updated_at from katello_repositories\" | su postgres -c 'psql foreman'" "katello_repositories"
+add_cmd "echo \"SELECT table_name, pg_size_pretty(total_bytes) AS total, pg_size_pretty(index_bytes) AS INDEX , pg_size_pretty(toast_bytes) AS toast, pg_size_pretty(table_bytes) AS TABLE FROM ( SELECT *, total_bytes-index_bytes-COALESCE(toast_bytes,0) AS table_bytes FROM (SELECT c.oid,nspname AS table_schema, relname AS TABLE_NAME, c.reltuples AS row_estimate, pg_total_relation_size(c.oid) AS total_bytes, pg_indexes_size(c.oid) AS index_bytes, pg_total_relation_size(reltoastrelid) AS toast_bytes FROM pg_class c LEFT JOIN pg_namespace n ON n.oid = c.relnamespace WHERE relkind = 'r') a) a order by total_bytes DESC\" | su postgres -c 'psql foreman'" "db_table_size"
```

Comment:
What are these two doing?

Reply:
~~~
 id |                           name                            | checksum_type |                                     relative_path                                      |         updated_at         
----+-----------------------------------------------------------+---------------+----------------------------------------------------------------------------------------+----------------------------
  3 | Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server     |               | Reference-Satellite/Library/content/dist/rhel/server/7/7Server/x86_64/os               | 2018-03-21 14:17:29.762139
  1 | Red Hat Satellite Tools 6.3 for RHEL 6 Server RPMs x86_64 | sha256        | Reference-Satellite/Library/content/dist/rhel/server/6/6Server/x86_64/sat-tools/6.3/os | 2018-03-21 14:19:09.670376
  2 | Red Hat Satellite Tools 6.3 for RHEL 7 Server RPMs x86_64 | sha256        | Reference-Satellite/Library/content/dist/rhel/server/7/7Server/x86_64/sat-tools/6.3/os | 2018-03-21 14:20:42.911033
  4 | Red Hat Enterprise Linux 6 Server RPMs x86_64 6Server     | sha1          | Reference-Satellite/Library/content/dist/rhel/server/6/6Server/x86_64/os               | 2018-03-21 17:14:05.638018
~~~